### ISSUE TITLE
RS819 and DSM 6.2.4 missing

### ISSUE THREAD

**Issue by xpunkt** [szyb/synopackage_dotnet#176]
in https://search.synopackage.com/info i cant select right models and version to find packages for

in the sources urls there is many invalid urls that cant be added, but still listed as active :/

sorry if i reported it in incorrect place, then i just like to know where to report

**Comment by szyb** [comment]
Hi,

In fact RS819 is missing. I will add it.
For DSM 6.2.4 there is an issue #175

@xpunkt  Could you point out which sources are invalid? All active sources are working fine (from my persprective)

**Comment by xpunkt** [comment]
http://synology.nzbusenet.com/

**Comment by szyb** [comment]
I investigated the problem with nzbusenet. Here is what I found:

- the source server responds with invalid JSON. They should fix it
- more deep: the problem is with the package's description for "plexpy-custom". The description is: "PlexPy Custom allows you to run the fork of your choice of PlexPy. You will be asked for the fork\'s git URL during install." and the character ' is the real problem
- Synopackage uses different JSON parser than DSM and this issue is ignored and shown as expected on the website

Summarize: This source server is invalid for DSM but it works in Synopackage. I will leave it as active, because this might be the easiest way to download packages from this source. Other option is to visit their website and download them manually.
